pinecone_text/sparse/splade_encoder.py

```diff
@@ -1,6 +1,20 @@
 from typing import List, Union
 
-import torch
+try:
+    import torch
+except (OSError, ImportError) as e:
+    raise ImportError(
+        "Failed to import torch. Make sure you install pytorch extra dependencies by running:"
```

Comment:
@acatav please note that this error would print out in one long line.  

If you want it to break it to lines - either use a `"""` block (like we do for prompts), or add explicit `\n` at the end of each string.